src/components/ComparisonContainer.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { connect } from 'react-redux'
+import interestJs from 'interestjs'
+
+import SectionWrapper from './commons/SectionWrapper'
+import Chart from './Chart'
+
+const BuyInput = ({ rent, buy, years, interest }) => {
+  const annualRentValue = 12 * rent
+  const { sum: rentTotal } = interestJs(annualRentValue, years, interest)
```

Comment:
Esse cálculo não deveria ser responsabilidade do componente de UI e sim do reducer que é onde deveria residir as regras de negócio da aplicação.

Além disso, acho que você não entendeu o problema corretamente. O cálculo de juros é aplicado ao valor de COMPRA (e não aluguel) se o comprador fosse financiar tal valor em X anos. Ou seja, se eu for financiar um imóvel de R$1.000.000 e os juros são 10% ao ano durante 30 anos (30 * 12 meses), eu vou pagar um valor total de R$1.852.000. Se eu alugar esse apartamento pelo mesmo período (30 * 12 meses), pagando R$5.000 por mês, eu pagarei R$1.800.000. Ou seja, nesse exemplo, vale mais a pena eu alugar do que comprar. Era isso que a ferramenta deveria informar.